src/schema/objects/metadata.yaml

```diff
@@ -574,7 +574,7 @@ Delimiter:
   name: Delimiter
   display_name: Delimiter
   description: |
-    If a column may be interpreted as a list of values, the character that
+    If rows in a column may be interpreted as a lists of values, the character that
```

Comment:
This might be better as:
"If a column entry may be interpreted as..."

Also, should you mention here that the default is a comma (since you are not requiring the Delimiter to be given?

Reply:
Delimiter will from now on apply to **any** column.